cmd/flags/flags.go

```diff
@@ -55,6 +54,7 @@ func SetGlobalFlags(flags *flag.FlagSet) *GlobalFlags {
 	flags.StringVarP(&globalFlags.Namespace, "namespace", "n", "", "The kubernetes namespace to use")
 	flags.StringVar(&globalFlags.KubeContext, "kube-context", "", "The kubernetes context to use")
 	flags.StringSliceVar(&globalFlags.Vars, "var", []string{}, "Variables to override during execution (e.g. --var=MYVAR=MYVALUE)")
+	flags.StringVarP(&globalFlags.KubeConfig, "kubeconfig", "k", "", "kubeconfig path")
```

Comment:
Please don't create a shorthand flag, and use the description `The kubeconfig path to use`